### PR TITLE
THU/show_powerbi_errors

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

File: src/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js [hunk 1]
```diff
@@ -11,12 +11,51 @@ import { useTranslation } from 'react-i18next';
...
     height: '100%',
     width: '100%'
+  },
+  errorContainer: {
+    height: '50px',
+    width: '100%',
+    position: 'absolute',
+    textAlign: 'center',
+    padding: '5px 0',
+    backgroundColor: theme.palette.error.dark,
```
Comment (COSMO-VMI) [inline src/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js:29]: I can't get to find the color used for wrong scenario names message, but I'd like both of them to have the same centralized value if you can. Maybe that's already the case.

**Review by COSMO-VMI — CHANGES_REQUESTED** [review]
Just a remark about error color. LGTM, GG

File: src/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js (hunk 1 above)
Comment (csm-thu) [reply src/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js:29]: The color for the scenario name errors is probably a default color right now.
When we want to unify the colors, using the `theme.palette.error.[dark/light/main]` value will be the proper way to do that :+1:

Comment (COSMO-VMI) [reply src/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js:29]: LGTM then :)

**Review by COSMO-VMI — APPROVED** [review]
LGTM, GG

**Pull request merged**